### PR TITLE
chg:dev:#88:Move Quark DDL statements from Calcite fork. 

### PR DESCRIPTION
**Pull request by vrajat**
DDL statements have now been moved from the Qubole's calcite fork. Refer to qubole/incubator-calcite#28 for the corresponding PR.

### CODE REVIEW

File: optimizer/src/main/codegen/includes/compoundIdentifier.ftl [hunk 1]
```diff
@@ -0,0 +1,34 @@
+<#--
+// Licensed to the Apache Software Foundation (ASF) under one or more
```
Comment (amoghmargoor) [inline optimizer/src/main/codegen/includes/compoundIdentifier.ftl]: Is this file required ?

**Comment by amoghmargoor** [comment]
LGTM ... So just for my information: Are we trying to unpack Parser.jj from calcite core  and add it to the generated sources ?

File: optimizer/src/main/codegen/includes/compoundIdentifier.ftl (hunk 1 above)
Comment (vrajat) [inline optimizer/src/main/codegen/includes/compoundIdentifier.ftl]: No. I am going to remove. Testing now.

**Comment by vrajat** [comment]
Yes. Parser.jj from calcite has hooks or variables for FMPP (which is a template engine). So you have to get the file and along with config.fmpp which specifies how to subsitute the vars, pass it into the FMPP engine. Out comes a customized parser.jj. From there, generate the parser (QuarkParserImpl) and create the parser instead of the stock Calcite parser.
